### PR TITLE
fix: topology spread timeout

### PR DESCRIPTION
**Pull request by szuecs**
disable: set to 0
defaults: 7m
change via CR

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -68,8 +68,8 @@ spec:
         prometheus.io/path: /metrics
         prometheus.io/port: "9911"
         prometheus.io/scrape: "true"
-{{- if eq .Cluster.ConfigItems.skipper_topology_spread_enabled "true" }}
-        zalando.org/topology-spread-timeout: 7m
+{{- if and (eq .Cluster.ConfigItems.skipper_topology_spread_enabled "true") (ne .Cluster.ConfigItems.skipper_topology_spread_timeout "0"}}
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/deployment.yaml]:
```suggestion
{{- if and (eq .Cluster.ConfigItems.skipper_topology_spread_enabled "true") (ne .Cluster.ConfigItems.skipper_topology_spread_timeout "0")}}
```

Was missing a `)`

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: thanks

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**